src/client/js/otp/config.js

```diff
@@ -174,7 +174,8 @@ otp.config = {
     	title: 'Icon Legend',
     	content: '<p><img src="images/locationSpot.svg" height="15" width="15"> : Current Location<br>\
     		<img src="images/busStopButton.png" height="15" width="15"> : BullRunner Bus Stop<br>\
-    		<img src="images/stop20.png" height="15" width="15"> : HART Bus Stop<br>'      	
+    		<img src="images/stop20.png" height="15" width="15"> : HART Bus Stop<br> \
+    		<p> We would like to acknowledge the support and funding assistance provided by the USF Student Green Energy Fund, Center for Urban Transportation Research, and Florida Department of Transportation.' 

```

Comment:
It looks like this is missing two closing </p> tags (before the ')  Otherwise, seems to be good